nixos/doc/manual/contributing-to-this-manual.xml

```diff
@@ -14,9 +14,6 @@ xlink:href="https://github.com/NixOS/nixpkgs">Nixpkgs</link> repository.
 <screen>
 <prompt>$ </prompt>cd /path/to/nixpkgs
 <prompt>$ </prompt>nix-build nixos/release.nix -A manual.x86_64-linux
+<prompt>$ </prompt>xdg-open ./result/share/doc/nixos/index.html
```

Comment:
I don't know if I like this change, since this would only work in Linux OS, and even them not all of them (not necessary everyone has `xdg-open` installed). macOS users are out of lucky.

Reply:
@thiagokokada well, this is the NixOS manual, and as such this is linux only. or do i misunderstand something here?

Reply:
my reasoning was that having something that works at most places, or can be made to work easily, is much better than not having something that can be easily copy-pasted.

otherwise it's just a relative path that the browser cannot even open in its URL bar when copy-pasted.

Reply:
Well, for example I could open in a browser terminal like Elinks (but my default browser is a graphical one). BTW, I said the part of macOS because someone maybe working in improving the NixOS manual from their macs.

I really prefer how it was before, because you don't have to assume anything about the user setup.

Reply:
ok, i'm new here, so i won't push this, i'll revert that part.

i still believe, though, that it's an improvement. people can still just ignore the xdg-open and copy-paste the ./result/ part, just like before.

and besides, the entire manual seems to be built assuming `x86_64-linux` in the previous line. that's much more platform dependent than the new xdg-open. it would be nice to get rid of that. e.g. by some `uname -foo`, or something like that. i was surprised by that when i first built the manual, but luckily it was matching my host.

Reply:
another suggestion: replace `xdg-open` to `[preferred browser]`.

Reply:
> i still believe, though, that it's an improvement. people can still just ignore the xdg-open and copy-paste the ./result/ part, just like before.

But now you're assuming that people know what `xdg-open` does. If they don't know that this is a "program to open files", they may even assume that this is something related to the manual build process.

> another suggestion: replace xdg-open to [preferred browser].

I quite like how it is, it is very clear what is happening. Even preferred browser is not ideal considering some of the more strange setups out there (Emacs for example).

BTW, I am just a collaborator of some reviews here, my word is not final so if you want to keep `xdg-open` and wait for another reviewer it is fine too.